### DISCUSSION TITLE
How to refuse new inbound channels?

### DISCUSSION THREAD

**Discussion by DerEwige** [ACINQ/eclair#2375] [Node operators]
I try to maintain a balanced routing node.
For this the inbound and outbound liquidity of my public channels should be in equilibrium if possible.
I would like to refuse new inbound channels if my node has already more than 60% of inbound liquidity.
But there does not seam to be any switch in the config that allows this.

Can I just set “min-public-funding-satoshis” to a value larger than “max-funding-satoshis”?
Will this work or is will this give an invalid config exception?

**Comment by t-bast** [comment]
This is an area we could improve, ideally we'd have hooks to let plugins decide on-the-fly whether to accept an incoming channel or not, but that's not possible today. The only thing you can do is to raise `min-public-funding-satoshis` and `min-private-funding-satoshis` to be very high so that no-one opens channels to you, but that requires a restart of `eclair` to apply the config change.

> Will this work or is will this give an invalid config exception?

This should work.

**Reply by t-bast** [reply]
Note that we will probably wait for dual funding and liquidity ads to be implemented before providing those hooks. Once we have these two features, it will make more sense for various plugins with different strategies to exist to decide whether to accept a channel and how much funds to contribute to it (if any).
